Client/Frontend/UserContent/UserScripts/BraveSkusHelper.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+'use strict';
+
+window.__firefox__.includeOnce("BraveSkusHelper", function() {
+  function sendMessage(method_id, data) {
+    return webkit.messageHandlers.braveSkusHelper.postMessage({'securitytoken': '$<security_token>' ,'method_id': method_id, data: data});
```

Comment:
It won't make a difference because this script is only injected on `accounts.brave.com` which is a page owned by us.
Also, atm, pages can override `[[redacted]]` and so they can already retrieve the handler and the token.
I'm going to be fixing all of these already, but this one in particular is fine IMO because we own the page, and it will never be injected onto another page.

Reply:
@Brandon-T we inject the script only on *.brave.com hosts, but the handler is available on any page (as I mentioned on Slack)
I can call `webkit.messageHandlers.braveSkusHelper.postMessage()` on localhost (or any host) and it will execute. This is also true for all webkit handlers we expose in `.page` world, the fact that we inject javascript into particular domains doesn't limit the accessibility of the webkit handlers.
In this particular case, this is not an issue because we have a host check here:
https://github.com/brave/brave-ios/blob/235c59c5f6a165e2bf5d66bf90e6f9eb0642a3cd/Client/Frontend/Browser/BraveSkusScriptHandler.swift#L42

But in general, we can't rely on where we inject .js files.

Reply:
^ This host check is actually a good example of how we should protect all host-related handlers, such as Search, proposed Jitsi SDK helper, etc. Instead of relying on JS obscurity, we should add security checks to the native code.